### DISCUSSION TITLE
Add options for recipe settings in either the docker-compose environment or the app's setting options

### DISCUSSION THREAD

**Discussion by shimizurei** [mealie-recipes/mealie#2869] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I want to preface this with that I enjoy most things about this application and I appreciate all the hard work put into it. _But..._

I don't really like that I can't blanket set various settings for my mealie instance. Previously, you used to have a setting where you closed comments on all recipes (iirc). I don't care for comments; they're an eyesore to me. I think you can keep it as-is, but it would be nice to have docker environment vars or settings that are app-wide because I'm tired of having to re-open the recipe I just saved because I forgot I have to turn off comments and turn on nutrition. Things like locking the recipe and making it private are appropriate on a per-recipe basis, but not the nutrition setting, imo.

Again, no-knocks, just frustrations. Thank you for the hard work!

### Please provide a concise description of the feature that would resolve your issue.

Something like `SHOW_COMMENTS: FALSE` or `SHOW_NUTRITION: TRUE`.

### Please consider and list out some caveats or tradeoffs made in your design decision

None

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by boc-the-git** [comment]
Edit: I probably won't progress this in the short term, see my subsequent comment. Near-original comment remains intact here:

This seems reasonable. I personally don't use the comments, and it had never bothered me that they're there.. but by the same token I might as well not have them there!
I'm not certain if we have a precedent for whether these are more suited to an environment variable or a setting within the app.

I might look to work on this when time allows, but obviously a busy time of year.
The comments change might be [here](https://github.com/mealie-recipes/mealie/blob/mealie-next/frontend/components/Domain/Recipe/RecipePage/RecipePage.vue#L75), and I've not yet looked at the nutritional. Depending on what the actual implementation is, you'd probably want to also remove the per-recipe setting so it wasn't possible to set something that had no effect.

**Comment by boc-the-git** [comment]
From a quick test, there is already some options under baseurl/admin/manage/groups/ - of course applying the setting at a group level rather than the whole instance.

It doesn't add nutrition/disable comments on existing recipes, but _it does_ work for any recipes created after you change the settings.

So that's probably _good enough_, if not perfect.

**Reply by shimizurei** [reply]
I wouldn't have ever thought to click on the groups tab since I'm the only user... Thanks I'll see how it works out.

**Reply by boc-the-git** [reply]
Yea I was just searching the code and saw some hints that it existed as a group-level setting. Hopefully it serves you well!
